qubit-sdk/src/main/java/com/qubit/android/sdk/internal/placement/callback/PlacementCallbackConnectorImpl.kt

```diff
@@ -0,0 +1,59 @@
+package com.qubit.android.sdk.internal.placement.callback
+
+import com.qubit.android.sdk.internal.common.logging.QBLogger
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.http.POST
+import retrofit2.http.Url
+
+internal class PlacementCallbackConnectorImpl(
+    private val impressionUrl: String,
+    private val clickthroughUrl: String
+) : PlacementCallbackConnector {
+
+  companion object {
+    @JvmStatic
+    private val LOGGER = QBLogger.getFor("PlacementCallbackConnector")
+
+    private const val BASE_URL_PLACEHOLDER = "http://localhost/"
+  }
+
+  private val callbackApi = Retrofit.Builder()
```

Comment:
Is Retrofit so lightweight, that we can create it for every call for placements?
I have doubts about it.